### PR TITLE
Add polyserve note to the unit test guide

### LINKED ISSUE #1483: add section to unit test guide discussing polyserve / devtools
continuation of discussion started in PR #1474...

@TimvdLippe: 

A great unit testing guide, really liked to read it. One thing though I would like to see added is testing using `polyserve`. It allows the developer to use features of the dev tools (break points, local css changes, etc...) and you can inspect the actual HTML elements on the page. Of course, for a full test suite run `wct` is superior, but when I am developing my own unit test, I prefer to run `polyserve`. My 2 cents.

@kaycebasques:

So @TimvdLippe if I understand correctly, you're recommending adding something about using `polyserve` to locally run an element / unit test, and then using Chrome DevTools to inspect the element to help implement the unit test. Is that correct? 

@TimvdLippe:

Yes, so the current workflow I use when working on a unit test is:

1. Run `polyserve` on root-element folder (or on a specific user test)
* Open Chrome and see the unit tests run
* Add/modify unit tests
* Refresh the page and immediately get response

Whenever a test fails, I can click on stacktraces and put breakpoints using the dev tools to debug the behavior. It is also possible to view the rendered CSS and modify rules accordingly.

For me these two advantages favor `polyserve` above `wct` for developing unit tests. When all unit tests succeed I run the whole suite with `wct`.

Edit: Ow and another advantage is I can select elements and hit `$0` in the console. Then I can inspect the attributes of an object and modify them to see the results. All in all, the dev tools help me accelerate the development of unit tests tremendously.

### PR DESCRIPTION
**Pull request by TimvdLippe**
I hope this description is clear enough to understand the benefits of using polyserve. Any feedback regarding grammar or my English in general is greatly appreciated.

Fixes #1483

### CODE REVIEW

File: 1.0/tools/tests.md [hunk 1]
```diff
@@ -30,6 +30,12 @@ popular third-party tools, including:
 This guide refers to Web Component Tester as `wct`, which is its command 
 line interface.
 
+Note: You can also use [polyserve](https://github.com/PolymerLabs/polyserve)
+to run your test suite in the browser. This allows you to use all browser
```
Comment (robdodson) [inline 1.0/tools/tests.md]: "This allows you to use browser features like dev tools and HTML inspection to debug your unit tests."

File: 1.0/tools/tests.md [hunk 2]
```diff
@@ -30,6 +30,12 @@ popular third-party tools, including:
...
+features like the dev tools and HTML inspection to debug your unit test.
+You can simply open the corresponding HTML page which contains the test
```
Comment (robdodson) [inline 1.0/tools/tests.md]: "You can then open the corresponding HTML page..."

**Comment by TimvdLippe** [comment]
Thanks for the review @robdodson ! I have updated the text accordingly.

**Comment by robdodson** [comment]
@arthurevans @kaycebasques do you feel ok on this?

**Comment by kaycebasques** [comment]
I was thinking more of a small step-by-step section. I'll have a look this week.

**Comment by arthurevans** [comment]
I wouldn't put it in the intro. I'd be inclined to add a separate section at the end of the Quick Start section, like:

    ### Run tests interactively with Polyserve

And as Kayce said, the last sentence should be replaced with a set of steps.

**Comment by TimvdLippe** [comment]
My reasoning was that as the whole guide uses `wct`, only a quick mention would fit. I didn't want to be too intrusive in the guide. Of course a longer explanation regarding Polyserve would be very nice.

@kaycebasques do I understand correctly you will take over the PR to create the step-by-step section?

**Comment by kaycebasques** [comment]
Hi @TimvdLippe,

Please add a section at the end of the Quick Start (per @arthurevans suggestion above) and move your content there. And thanks for the contribution!

**Comment by TimvdLippe** [comment]
@kaycebasques I have moved it to the end of the Quick Start section.Glad to help out =]

**Comment by arthurevans** [comment]
LGTM. Thanks!

**Pull request merged**